ovos_plugin_common_play/ocp/utils.py

```diff
@@ -56,3 +57,34 @@ def create_desktop_file():
     dst_icon = join(icon_path, "OCP.png")
     if not isfile(dst_icon):
         shutil.copy(src_icon, dst_icon)
+
+
+def module_property(func):
```

Comment:
this is a good one to join classproperty in ovos_utils for usage in more places

Reply:
Agree. I wanted to figure out the config bug before adding to utils